coopr-docs/docs/source/release-notes/release-0.9.9.rst

```diff
@@ -25,14 +25,135 @@
 Welcome to the Coopr 0.9.9 release. Release notes with links to issue details can be seen
 from the github project at https://github.com/caskdata/coopr/releases.
 
-Release Theme: 
------------------------------
-
 Release Highlights
 ------------------
-  * 1
-  * 2
+  * Docker support, ability to deploy Docker services and manage Docker containers
+  * Provider plugin for Digital Ocean added
+  * New UI, rewritten to be more modular and extensible
+  * CLI tool, allows interaction via simple commands
+  * Split Server APIs into internal and external, served on different ports
```

Comment:
Splits the Server APIs into internal and external APIs, served on different ports

(at first, I thought it was a new type of "Split Server"...)